src/main/java/storage/FileManager.java

```diff
@@ -66,6 +67,36 @@
 
     private static Logger logger = Logger.getLogger(FileManager.class.getName());
 
+    /**
+     * Recreates the workout file.
+     * @throws IOException      If the application is unable to create the workout file.
+     */
+    public void deleteAndRecreateWorkoutFile() throws IOException {
+        File workoutFile = new File(String.valueOf(getWorkoutFilePath()));
+        workoutFile.delete();
+        workoutFile.createNewFile();
```

Comment:
Would it be possible to call the existing `createWorkoutFile()` method to create the file? That way some redundant code can be removed. Similar for the other files.

In addition, perhaps some `Logger` calls can be done to log that the file has been deleted?

Reply:
Yes, it is possible, but Im also trying to delete the file before creating it, perhaps you mean I remove `workoutFile.createNewFile();` and replace with `createWorkoutFile()` ?

Reply:
Yup, that's what I meant, since we already have a dedicated method create the file. So you keep the `workoutFile.delete()` and replace line 77 with `createWorkoutFile()`.

Reply:
Alright, will do that, thank you for you suggestions!